portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -40,7 +53,11 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String comment = request.getParameter("comment");
-    comments.add(comment);
+
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("content", comment);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Make this an instance variable and instantiate in constructor? That way you can get rid of this in both methods.

Reply:
I see. I'll modify this in the next PR as well. Thanks.